src/main/java/msjava2lesson/reverselinkedlist/ListNode.java

```diff
@@ -0,0 +1,19 @@
+package msjava2lesson.reverselinkedlist;
+
+public class ListNode{
+
+    public int val;
+
+    public ListNode next;
```

Comment:
by default, in Java we we have `private` access modifier for fields
```suggestion
    private int val;

    private ListNode next;
```

Reply:
All fields are public for that class - I use this fields in code of Main class without setters.